tests/checkers/unittest_unicode/__init__.py

```diff
@@ -0,0 +1,85 @@
+from contextlib import contextmanager
+from pathlib import Path
+
+import pylint.interfaces
+import pylint.testutils
+
+CODEC_AND_MSG = [
+    ("utf-8", tuple()),
+    (
+        "utf-16",
+        (
+            pylint.testutils.MessageTest(
+                msg_id="invalid-unicode-codec",
+                confidence=pylint.interfaces.HIGH,
+                # node=module,
```

Comment:
It looks like dead code, is it ?

Reply:
Nope. Its left as @DanielNoord wanted to include nodes but this led to Pylint crashing at the moment. So I added and commented the node for future reference. But I didn't want to add a TODO everywhere....
So it sits there by intention.

Reply:
:+1: I'm resolving.